lib/host/wasi/inode-win.cpp

```diff
@@ -330,6 +330,26 @@ SymlinkPriviledgeHolder SymlinkPriviledgeHolder::Holder;
 
 } // namespace
 
+namespace {
+static winapi::HANDLE_ getWindowsHandle(int32_t Fd, winapi::DWORD_ StdHandleId,
+                                        std::string_view Name) {
```

Comment:
Why do we need `Name` here? It seems like it's going to be used inside the warning only.